web/frontend/src/api/movement-sensor.ts

```diff
@@ -1,193 +1,61 @@
-import { type Client, movementSensorApi } from '@viamrobotics/sdk';
 import { rcLogConditionally } from '@/lib/log';
+import { type Client, MovementSensorClient } from '@viamrobotics/sdk';
 
 export const getProperties = async (robotClient: Client, name: string) => {
-  const req = new movementSensorApi.GetPropertiesRequest();
-  req.setName(name);
-
-  rcLogConditionally(req);
-
-  const response =
-    await new Promise<movementSensorApi.GetPropertiesResponse | null>(
-      (resolve, reject) => {
-        robotClient.movementSensorService.getProperties(req, (error, res) => {
-          if (error) {
-            reject(error);
-          } else {
-            resolve(res);
-          }
-        });
-      }
-    );
-
-  return response?.toObject();
+  const client = new MovementSensorClient(robotClient, name, {
+    requestLogger: rcLogConditionally,
+  });
+  return client.getProperties();
 };
 
 export const getOrientation = async (robotClient: Client, name: string) => {
-  const req = new movementSensorApi.GetOrientationRequest();
-  req.setName(name);
-
-  rcLogConditionally(req);
-
-  const response =
-    await new Promise<movementSensorApi.GetOrientationResponse | null>(
-      (resolve, reject) => {
-        robotClient.movementSensorService.getOrientation(req, (error, res) => {
-          if (error) {
-            reject(error);
-          } else {
-            resolve(res);
-          }
-        });
-      }
-    );
-
-  return response?.toObject().orientation;
+  const client = new MovementSensorClient(robotClient, name, {
+    requestLogger: rcLogConditionally,
```

Comment:
[aside] Do you know if these clients expensive to construct? I traced the trail down through connect-es PromiseClient but didn't go much further.